.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["env", "stage-2"],
+  "plugins": []
+}
```

Comment:
Why is this here?  also, why is the plugins array empty?

Reply:
I added these here, so I will not have to be repeating the `env` and `stage-2` whenever I make use of the babel command in scripts, will remove the empty plugins array